tests/unit/test_wheel.py

```diff
@@ -268,7 +268,29 @@ def test_std_install(self, data, tmpdir):
             scheme=self.scheme,
             req_description=str(self.req),
         )
-        self.assert_installed()
+        self.assert_installed(0o644)
+
+    @pytest.mark.parametrize("user_mask, expected_permission", [
+        (0o27, 0o640)
+    ])
+    def test_std_install_with_custom_umask(self, data, tmpdir,
+                                           user_mask, expected_permission):
+        """Test that the files created after install honor the permissions
+        set when the user sets a custom umask"""
+
+        prev_umask = os.umask(0)
+        try:
+            os.umask(user_mask)
```

Comment:
```suggestion
        prev_umask = os.umask(user_mask)
        try:
```

Reply:
Makes sense as per [os.umask](https://docs.python.org/3/library/os.html#os.umask) . Fixed in latest commit